packages/cfd/src/Containers/cfd-financial-stp-real-account-signup.tsx

```diff
@@ -159,7 +159,7 @@ const CFDFinancialStpRealAccountSignup = (props: TCFDFinancialStpRealAccountSign
         index: number,
         value: { [key: string]: string | undefined },
         setSubmitting: TSetSubmiting,
-        is_dirty: boolean = true
```

Comment:
this gave a warning: Type boolean trivially inferred from a boolean literal, remove type annotation.